package ua.goit.module.tenth.task5;

public class FirstException extends Exception{
}
